security/security-spring/src/main/java/org/cbioportal/security/spring/authentication/social/PortalUserDetailsService.java

```diff
@@ -69,6 +70,9 @@
 
     @Autowired
     private SecurityRepository securityRepository;
+    
+    @Value("${authorization:}")
```

Comment:
Again, I am not sure why this class was modified.  It is only used in applicationContext-security.xml in the "social_auth" and "googleplus" profiles.  So AUTHORIZATION must be true.